lib/data/model/response_models/series.dart

```diff
@@ -0,0 +1,22 @@
+class Series {
+  final int? id;
+  final String? title;
+  final String? description;
+  final String? thumbnailPath;
+  final String? thumbnailExtension;
+
+  const Series(
+      {required this.id,
+      required this.title,
+      required this.description,
+      required this.thumbnailPath,
+      required this.thumbnailExtension});
+
+  String getThumbnailUrl(String size) {
+    String thumbnailUrl = "";
```

Comment:
`String thumbnailUrl = "";` does not make sense right? So in this case returning `null` is more siutable.

Reply:
in case when thumbnailUrl == null widget FadeInImage show mistake. image parametr can't be null. It also can't be empty string. because when image loading it show placeholder, when loading is done, but imageUrl == emptystring or another wrong url I catch an error and show it in ui.

Reply:
maybe in case when thumbnailUrl equal null or empty string we can add check for it and add widget with image from assets or no widget (when null). i dont think about it before beacouse it not obviously for me.

Reply:
If both urls empty/null are invalid is better to return null and before passing it as an argument do the null-check.

Reply:
done